inductiva/resources/__init__.py

```diff
@@ -0,0 +1,3 @@
+#pylint: disable=missing-module-docstring
+from .machines import MachineGroup
+from .resources import *
```

Comment:
we shouldn't import things like this. By doing this you'll also be exposing things that you use in resources.py (e.g. absl.logging and things like that)

Reply:
Thanks for pointing that out, I will fix it now.